Experiments/Experiments/FeatureFlag.swift

```diff
@@ -53,4 +53,8 @@ public enum FeatureFlag: Int {
     /// Displays the Orders tab in a split view
     ///
     case splitViewInOrdersTab
+
+    /// Displays the option to edit a coupon
+    ///
+    case couponEditing
```

Comment:
For Coupons M2 we enable both editing and deleting coupons - should we keep the same feature flag for both features or is it preferred to have separate flags? I'm using one for both in #6496 actually.

Reply:
I believe that's OK to mantain two separate feature flags, we can enable them at the same time. Otherwise could be more clean to rename the existing feature flag to `couponsM2` or something like that, since we are going to integrate different functionalities under the same flag.